bin/crew

```diff
@@ -40,9 +40,10 @@ DOC = <<~DOCOPT
   Usage:
     crew build [options] [-k|--keep] <name> ...
     crew const [options] [<name> ...]
-    crew deps [options] [-t|--tree] [-b|--include-build-deps] [--exclude-buildessential] <name> ...
+    crew deps [options] [--deep] [-t|--tree] [-b|--include-build-deps] [--exclude-buildessential] <name> ...
```

Comment:
Add `-d|--deep` short option too?

Reply:
There is already a `-d` option for crew, so I don't want to create a conflict.